save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt

```diff
@@ -243,6 +265,67 @@ class SandboxView : AbstractView<SandboxViewProps, SandboxViewState>(false) {
         }
     }
 
+    private fun uploadTests() {
+        scope.launch {
+            setState {
+                isUploading = true
+            }
+
+            postTestAsText("TEST", "test", state.codeText)
+            postTestAsText("TEST_RESOURCE", "save.toml", state.configText)
+            postTestAsText("TEST_RESOURCE", "setup.sh", state.setupShText)
+
+            setState {
+                isUploading = false
+            }
+        }
+    }
+
+    private fun loadTests() {
+        scope.launch {
+            val newCodeText = getTestAsText("TEST", "test", codeExample)
+            val newConfigText = getTestAsText("TEST_RESOURCE", "save.toml", configExample)
+            val newSetupShText = getTestAsText("TEST_RESOURCE", "setup.sh", setupShExample)
+
+            setState {
+                codeText = newCodeText
+                configText = newConfigText
+                setupShText = newSetupShText
+            }
+        }
+    }
+
+    private suspend fun postTestAsText(
+        type: String,
```

Comment:
`SandboxStorageKeyType` instead of string?

Reply:
`SandboxStorageKeyType` is not available in frontend. I don't want to move it to common place to be able to extract sandbox in future.
Will split it to two endpoints: `upload-test-as-text` and `upload-test-resources-as-text`